docs/pages/database-access/guides/rds.mdx

```diff
@@ -254,7 +254,7 @@ Attach created policy and permission boundary to the IAM identity your Teleport
 database agent will be using. For example, if the agent runs as an IAM user:
```

Comment:
As in the Redshift guide, I think this needs a sentence indicating what this screenshot shows.

Reply:
new file `docs/pages/includes/database-access/attach-iam-policies.mdx`